lib/screens/collective/perspectives/expression_feed.dart

```diff
@@ -45,7 +51,10 @@ class _ExpressionFeedState extends State<ExpressionFeed> {
                 //  between layouts creates an issue.
                 const SliverToBoxAdapter(),
                 if (state is CollectivePopulated)
-                  CollectivePopulatedList(state),
+                  CollectivePopulatedList(
+                    state,
+                    deleteExpression: _removeExpression,
```

Comment:
You don't have to pass this through constructor. Just get the Bloc where it's needed.

Reply:
That was the original plan I had for implementing this but the same widget is used in 3 different places all using a different bloc. Ideally, it would've been better to make a single call like `context.bloc<ExpressionBloc>().add(Delete())` but unfortunately, this is not possible with the way things are currently structured. 

I opted to go with the current approach for the time being since it allows us to handle a delete for all supported bloc (Packs, Den and Collective).

Reply:
Sure, no problem